### PR TITLE
Print language code after web server address info

### PR DESCRIPTION
**Pull request by ilmari-lauhakangas**
With dozens of different languages, it is nice to see which port maps to which language in local development.

### CODE REVIEW

File: commands/server.go [hunk 1]
```diff
@@ -927,7 +927,11 @@ func (c *serverCommand) serve() error {
 
 	for i := range baseURLs {
 		mu, listener, serverURL, endpoint, err := srv.createEndpoint(i)
-		var srv *http.Server
+		var srv *http.Server	
+		langMsg := ""
+		if roots[i] != "" {
+			langMsg += fmt.Sprintf(". Language: %s", roots[i])
+		}
```
Comment (jmooring) [inline commands/server.go:934]:
```suggestion
```

File: commands/server.go [hunk 2]
```diff
@@ -954,7 +958,7 @@ func (c *serverCommand) serve() error {
 			mu.HandleFunc(u.Path+"/livereload.js", livereload.ServeJS)
 			mu.HandleFunc(u.Path+"/livereload", livereload.Handler)
 		}
-		c.r.Printf("Web Server is available at %s (bind address %s)\n", serverURL, c.serverInterface)
+		c.r.Printf("Web Server is available at %s (bind address %s)%s\n", serverURL, c.serverInterface, langMsg)
```
Comment (jmooring) [inline commands/server.go:961]:
```suggestion
		c.r.Printf("Web Server is available at %s (bind address %s) %s\n", serverURL, c.serverInterface, langMsg, roots[i])
```

**Review by jmooring — COMMENTED** [review]
I think this is a good idea, and has been requested before. But I think all we need is:

Screenshot-20230910110936-770x72

**Comment by ilmari-lauhakangas** [comment]
Sorry about the trailing whitespace :(

**Pull request merged**